###############################################################################
#
# Copyright (c) 2009, 2023, Oracle and/or its affiliates.
#
# Permission is hereby granted, free of charge, to any person obtaining a
# copy of this software and associated documentation files (the "Software"),
# to deal in the Software without restriction, including without limitation
# the rights to use, copy, modify, merge, publish, distribute, sublicense,
# and/or sell copies of the Software, and to permit persons to whom the
# Software is furnished to do so, subject to the following conditions:
#
# The above copyright notice and this permission notice (including the next
# paragraph) shall be included in all copies or substantial portions of the
# Software.
#
# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
# DEALINGS IN THE SOFTWARE.
#

Note that due to us not packaging /usr/lib/X11/fs/config as an editable file,
upgrades will overwrite it, and customizations need to happen in the /etc
counterpart instead.

This is specific to Solaris and not suitable for upstream.

diff -urp -x '*~' -x '*.orig' config/config.cpp config/config.cpp
--- a/config/config.cpp	2010-08-06 18:40:29.000000000 -0700
+++ b/config/config.cpp	2011-01-28 18:01:17.331211005 -0800
@@ -1,6 +1,13 @@
 # X font server configuration file
 # See xfs(__appmansuffix__) man page for more information.
 
+XCOMM   DO NOT EDIT /usr/lib/X11/fs/config directly.
+XCOMM
+XCOMM   /usr/lib/X11/fs/config is a factory-default file and will
+XCOMM   be unconditionally overwritten upon subsequent installation.
+XCOMM   Before making changes to the file, copy it to the system configuration 
+XCOMM   location, /etc/X11/fs/config and edit it there.
+    
 clone-self = on
 use-syslog = off
 catalogue = DEFAULTFONTPATH
